addons/crm_livechat/static/tests/mock_server/mock_models/discuss_channel.js

```diff
@@ -14,7 +14,7 @@ const discussChannelPatch = {
         this.env["bus.bus"]._sendone(serverState.partnerId, "discuss.channel/transient_message", {
             body: `
                     <span class="o_mail_notification">
-                        Create a new lead: <a href="#" data-oe-model="crm.lead" data-oe-id="${leadId}">${leadName}</a>
+                        Created a new lead: <a href="#" data-oe-model="crm.lead" data-oe-id="${leadId}">${leadName}</a>
```

Comment:
IMO this is still poorly worded. I would write _"New lead has been created: ..."_

Reply:
Ok this was already the wording in python, so it just matches it. Let's keep like this for now.